android/build.gradle

```diff
@@ -478,4 +466,16 @@ tasks.whenTaskAdded { task ->
         task.dependsOn(extractAARHeaders)
         task.dependsOn(extractSOFiles)
     }
-}
\ No newline at end of file
+}
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+                groupId = 'com.github.wordpress-mobile'
+                artifactId = 'react-native-reanimated'
+            }
+        }
+    }
+}
```

Comment:
The Android project we're publishing is located in `android-npm` ([reference](https://github.com/wordpress-mobile/react-native-reanimated/blob/ae08ebd3ad6ac863dbde916bf80050ba26a66506/android-npm/build.gradle#L21-L32)), this one is only used for generating the AAR (`android-npm/react-native-reanimated-66-hermes.aar`) and is invoked when running `./createNPMPackage.sh`. So, in theory, we don't need to add part.

Reply:
Oh cool, I am going to revert that change then, thanks for letting me know @fluiddot ! 🙇

Reply:
@ParaskP7 Thanks for addressing this 🙇 .